cfme/tests/v2v/test_csv_import.py

```diff
@@ -269,17 +253,36 @@ def test_csv_archived_vm(appliance, infra_map, archived_vm):
     """Test csv with archived vm name
     Polarion:
         assignee: ytale
-        caseimportance: high
         caseposneg: positive
-        testtype: functional
         startsin: 5.10
         casecomponent: V2V
         customerscenario: true
         initialEstimate: 1/8h
-        subcomponent: RHV
-        upstream: yes
     """
     content = "Name\n{}".format(archived_vm)
     error_msg = "VM is inactive"
     assert import_and_check(appliance, infra_map, error_msg,
                             content=content, table_hover=True)
+
+
+@pytest.mark.uncollectif(lambda provider: provider.one_of(RHEVMProvider))
+def test_csv_security_group_flavor(appliance, infra_map, valid_vm, provider):
+    """Test csv with openstack security group and flavor
+    Polarion:
+        assignee: ytale
+        caseposneg: positive
+        startsin: 5.10
+        casecomponent: V2V
+        customerscenario: true
+        initialEstimate: 1/4h
+    """
+    security_group = provider.data.security_groups.admin[1]
```

Comment:
What is this index for? What structure are you expecting in the provider data? Is this supposed to be a string or a map of values?

Reply:
we have two projects `admin` and `qe-auto` in OSP and each project have primary and secondary security group. I am accessing secondary group here from `admin` project which is suppose to be string.

Reply:
This needs to be described in the test, there's nothing here that describes those requirements.

Reply:
Updated with ``Test csv with "secondary" openstack security group and flavor``